react/game/src/app/screens/Login/components/LoginForm/index.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import { reduxForm } from 'redux-form';
+import ROUTES from '@constants/routes';
+import { Link } from 'react-router-dom';
+import { func } from 'prop-types';
+import Logo from '@components/Logo';
+
+import styles from './styles.scss';
+
+class LoginForm extends Component {
+  handleSubmit = event => {
+    event.preventDefault();
```

Comment:
This will be removed when i introduce Redux-forms, there is no need of that code, or that function, but that is just for testing that the form is working ok